pkgs/by-name/co/code-cursor/package.nix

```diff
@@ -8,11 +8,11 @@
 }:
 let
   pname = "cursor";
-  version = "0.43.0";
+  version = "0.43.5";
   appKey = "230313mzl4w4u92";
   src = fetchurl {
-    url = "https://download.todesktop.com/230313mzl4w4u92/cursor-0.43.0-build-24112423a8e6ct7-x86_64.AppImage";
```

Comment:
Could mirror to web.archive.org and add that URL too so if this dies the derivation still works, since it seems they don't stick around forever.

Reply:
The cursor team is setting up an archive of old builds for people to roll back to. I and several others made enough noise that they finally chose to do the right thing.

Reply:
@LunNova unfortunately, your comment is blocking merging. Can you approve, please?

Reply:
Wait why was my ordinary comment as review blocking approval? It wasn't "needs changes" so shouldn't impact that. :confused:

Reply:
It was entered as a recommended change (delete the URL plus some text). The person who keeps the "ready to merge" tags up to date sees this as "it's not ready yet"). 

Another way to solve this would have been to mark this comment as resolved.